ui/src/app/views/new-dataset-modal.spec.tsx

```diff
@@ -59,7 +60,7 @@ describe('NewDataSetModal', () => {
     expect(exportSpy).not.toHaveBeenCalled();
   });
 
-  it('should not export if export not clicked', async() => {
+  it('should not export if export  clicked', async() => {
```

Comment:
```suggestion
  it('should not export if export not clicked', async() => {
```

Reply:
This is because now export functionality is selected by default, if its un checked it would mean user does not want to export notebook

Reply:
Oh that makes sense. how about this then?

```suggestion
  it('should not export if export not checked', async() => {
```